vortex-proto/src/generated/vortex.dtype.rs

```diff
@@ -1,22 +1,18 @@
 // This file is @generated by prost-build.
-#[allow(clippy::derive_partial_eq_without_eq)]
```

Comment:
this was changed in https://github.com/tokio-rs/prost/pull/1115 so we now need to allow(clippy::nursery) in the module level